TEDU-52 Revamped the Home page topics (#176)
mautic/mautic-documentation: pages/01.Home/02.Contributing/docs.en.md

<!-- ## Table of Contents -->

<!--
Use this site to generate the TOC list elements:

- https://ecotrust-canada.github.io/markdown-toc/

remove the first two lines
-->

- [Get Started](#get-started)
- [Report Issues and Fix Bugs](#report-issues-and-fix-bugs)
- [Documentation Repository Structure](#documentation-repository-structure)
- [Style guide](#style-guide)
- [How to contribute](#how-to-contribute)
  * [Using Grav](#using-grav)
  * [Using Github](#using-github)
    + [Editing documentation using the command-line](#editing-documentation-using-the-command-line)
    + [Editing documents in the GitHub browser interface](#editing-documents-in-the-github-browser-interface)
- [Mautic specific information](#mautic-specific-information)
  * [Example domain references](#example-domain-references)
    + [Protocol scheme: http vs https](#protocol-scheme--http-vs-https)
  * [Linking to Release versions](#linking-to-release-versions)
    + [`latest` version](#-latest--version)
  * [References to _leads_ and _lead lists_](#references-to--leads--and--lead-lists-)
      - [reference note](#reference-note)

## Get Started

>  Although we support both Git and Grav for  documentation, we'd prefer contributions using Git because of the following reasons:

Author: Changed 'Github' to 'GitHub'.

   - The **`themes`** folder contains all the themes available for this repository. Each folder contains supporting files for a separate theme.

## Style Guide

For keeping contributions consistent, we encourage you to read our style guide which is a set of conventions that we follow while writing documentation for Mautic. You can find our style guide [here][style guide].

 Mautic documentation is written with [Markdown markup][markup], a simple but flexible text formatting language. The Markdown files are appended with the *.md* extension.

## How to contribute

### Using Grav

Previously Gitbook was used to manage the Mautic documentation. However, this became cumbersome to maintain and difficult for those who were not familiar with Git. After exploring various solutions, the Education Team decided that Grav would best meet the needs of the Community.
1. Go to the [the login page][mautic-docs-login]. Create a Mautic Community login, or log in with your existing forum/website account.
1. Send a message on the [Documentation slack channel][mautic-docs-slack] to have your account raised to 'editor' - get a Slack invite [here][mautic-slack-invite].

> 1. When logged in, click the **Edit page in Grav** link on the relevant page and make your changes.

Reviewer: We had to disable this feature for some reason I cannot recall, so I would suggest that folk use the UI

```suggestion
1. When logged in, go to the admin panel (ask the team for the link) and make your changes.
```

### Using Github

1. Fork this [GitHub][mautic-docs-github] repository.
1. Select a file to edit on your fork.
1. Make your commits.
1. Open a pull request to `base fork: mautic/mautic-documentation` with `base: master`.

> 1. Include and reference any [Issues][doc-issues] your Pull Request addresses.

Reviewer: ```suggestion
1. Include and reference any [Issues][doc-issues] your Pull Request addresses. Be sure to write 'closes #0001' if your PR will close an existing issue, so that they are automatically linked.
```

#### Editing documentation using the command-line

> To edit documents using the git command-line system, read this section.

Reviewer: ```suggestion
To edit documents using the Git command-line system, read this section.
```

Reviewer: You may want to tell folk about the Github CLI utility - makes working at the command line much easier! https://cli.github.com/

Author: I'll make a note about the GitHub CLI utility.

   - GitHub web interface - `base fork: mautic/mautic-documentation` and `base: master` at [GitHub][mautic-docs-github]
   - Use the [`hub`][hub] utility

    ````console
    hub pull-request
    ````

#### Editing documents in the GitHub browser interface

If you're unfamiliar with the Git command-line but still want to contribute to the Mautic documentation via Github, read this section.

Using *README.md* as an example:

1. [Fork][mautic-docs-fork] this repository under your account so you'll have permission to edit.
1. Select the *README.md* file . Refer to the [Mautic Documentation Repository Structure](#mautic-documentation-repository-structure) section. 
1. With the content of *README.md* visible, click on the pencil icon to begin editing the file.
1. After you've made a change, scroll down to the *Commit changes* form. Saving your change requires describing what was changed and why.
1. Before submitting your commit, select the box for *Creates a new branch* to start a new branch for your change. Name your branch something like `{yourusername}-revision-readme-file`

> 1. Select *Propose file change*

Reviewer: You may want to add that if they are changing more than one file they can then select the newly created branch to switch to the branch, and then repeat this process until they have made all the edits they need, before making the PR.

Author: Added this note:
If you are updating more than one file, then you can select the newly created branch to switch to the branch, and then repeat this process until you have made all the required edits, before creating a pull request.

> - _anonymous leads_ were renamed to _**visitors**_

[release-1.4.0]: <https://github.com/mautic/mautic/releases/tag/1.4.0>
```

</details>

<!-- markdown style links -->

[CONTRIBUTING]: <https://github.com/mautic/mautic-documentation/CONTRIBUTING.md>
[release-list]: <https://github.com/mautic/mautic/releases>
[release-latest]: <https://github.com/mautic/mautic/releases/latest>
[release-2.15.3]: <https://github.com/mautic/mautic/releases/tag/2.15.3>
[release-2.9.0]: <https://github.com/mautic/mautic/releases/tag/2.9.0>
[release-1.4.0]: <https://github.com/mautic/mautic/releases/tag/1.4.0>

[mautic-docs]: <https://docs.mautic.org>
[mautic-docs-login]: <https://docs.mautic.org/login>
[mautic-docs-github]: <https://github.com/mautic/mautic-documentation>
[mautic-docs-fork]: <https://github.com/mautic/mautic-documentation#fork-destination-box>
[mautic-doc-license]: <https://github.com/mautic/mautic-documentation/blob/master/LICENSE>
[doc-issues]: <https://github.com/mautic/mautic-documentation/issues>

[developer-docs]: <https://developer.mautic.org>
[developer-docs-github]: <https://github.com/mautic/developer-documentation>

[Mautic]: <https://mautic.org/>

> [mautic]: <https://mautic.org/>

Reviewer: We should not be duplicating link macros - let's pick one and update accordingly.

```suggestion
```
